c/alloc.c

```diff
@@ -453,7 +467,9 @@ char *safeMalloc2(int size, char *site, int *indicator){
     memset(res,0,size);
     *indicator = 0x884;
   } else{
-    printf("MALLOC failed, got NULL for size %d at site %s\n",size, site);
+    if (MALLOC_TRACE_LEVEL >= 1){
+      printf("MALLOC failed, got NULL for size %d at site %s\n",size, site);
```

Comment:
This just may not be needed at all, because you will know success by getting a return of NULL.